mdbenchmark/generate.py

```diff
@@ -134,18 +115,29 @@ def generate(name, gpu, module, host, min_nodes, max_nodes, time, list_hosts):
         directory = '{}_{}'.format(host, m)
         gpu_string = '.'
 
+        # here we switch between the md engines like gromacs and namd
+        # namd input files can have relative imports. To avoid issues when
+        # copying the files we currently only allow the use of absolute paths. 
+        engine = detect_md_engine(m)
+        if engine == namd:
+            click.secho('!!!ATTENTION!!!', bold=True)
+            click.echo("""This is an experimental NAMD support!
+All files must be in this directory.
+Parameter paths must be abslute in NAMD files!
+            """)
```

Comment:
Maybe rather add this into the generate function of the NAMD module.

Reply:
if one moves it to the namd generate function it will be shown x times for each folder which is generated..maybe a new util fxn? we'll probably need such a warning for other engines too.

Reply:
Will we keep this message when merging? We should not merge anything that is not _fully functional_? If we are sure that it is working, which the unit/integration test _should_ pledge for, then we can get rid of that message entirely. Also trying it out on the clusters would help reassure that everything is working :fire: